apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -140,6 +148,37 @@ const BookingPage = ({
     },
   });
 
+  const recurringMutation = useMutation(createRecurringBooking, {
+    onSuccess: async (responseData: any[]) => {
+      const { attendees, recurringEventId } = responseData[0];
+      const location = (function humanReadableLocation(location) {
+        if (!location) {
+          return;
+        }
+        if (location.includes("integration")) {
+          return t("web_conferencing_details_to_follow");
+        }
+        return location;
+      })(responseData[0].location);
+
+      return router.push({
+        pathname: "/success",
+        query: {
+          date,
+          type: eventType.id,
+          eventSlug: eventType.slug,
+          recur: recurringEventId,
```

Comment:
If the booking was from a recurring event type, we pass down the `recur` query parameter to have a different treatment in the UI.